docs/developers/datastore.md

```diff
@@ -1,12 +1,10 @@
 # Datapackage Registry Datastore
 
-The Datapackage Registry Datastore is a flat file datastore with source data stored in [Data Packages](http://specs.frictionlessdata.io/).
-
-There is not a great deal to document. The following notes are important to know:
+The Datapackage Registry uses flat file datastore to store uploaded [Data Packages](http://specs.frictionlessdata.io/).
 
 - The Datastore runs from S3, or, an object storage which supports the S3 API.
 - The Datastore is the point of truth for data, holding the raw data sources, and the [Data Package](http://specs.frictionlessdata.io/) that describes the sources.
```

Comment:
You don't need to keep linking Data Packages. Linking once at the top is enough. People reading this kind of technical document will know about data packages - or know how to find about them.